docs/frontend.md

```diff
@@ -53,10 +53,6 @@ eliminate the effort involved with applying correct formatting. As a reviewer, i
 debates over code style, since there is a consistent style being enforced through the adopted
 tooling.
 
-Prettier works reasonably well in combination with Airbnb's JavaScript standards. In the few cases
-where conflicts occur, formatting rules may be disabled to err toward Prettier conventions when an
-option is not configurable.
-
```

Comment:
should we state that our config is "inspired by Airbnb's" or have we diverged from that too much?

Reply:
Eh, yeah, I dunno that it's particularly useful to maintain the historical alignment or imply that we still seek to keep that alignment. In the future, I think we could also do to trim down the configuration a bit more, I just didn't want to change too much too quickly in this initial pull request.